Dockerfile.cpu

```diff
@@ -38,7 +38,7 @@ ENV LANG=C.UTF-8
 
 COPY requirements.txt .
 COPY requirements_test.txt .
-COPY scripts/ scripts/
+COPY scripts/install_requirements.sh scripts/install_requirements.sh
```

Comment:
I added better comments in https://github.com/allenai/allennlp/pull/107/files--I'll make sure those get merged in even if we don't run tests inside our Docker build.